### ISSUE TITLE
when importing pddlgym, get invalid syntax error

### ISSUE THREAD

**Issue by aymeric75** [tomsilver/pddlgym#91]
In a python file located at pddlgym folder, I do 

`import pddlgym`

and I get 
```

  File "/pddlgym/pddlgym/__init__.py", line 35
    **other_args}),
     ^
SyntaxError: invalid syntax
```

**Comment by tomsilver** [comment]
Is it possible that you're running an earlier version of Python? If that's not it, see if you can make a minimal reproducible example using a fresh install of pddlgym.

**Comment by aymeric75** [comment]
yes I don't why but my python version changed to 2.7, solved now with 3.10, thanks
